### ISSUE TITLE
Accessibility: the link to accept the starter kit has the link text of "Loading"

### ISSUE THREAD

**Issue by lynnewritescode** [umbraco/Umbraco-CMS#6383]
Reproduction
------------
### Bug summary
On the page to select if you want a starter kit or not, the link to accept the starter kit has the link text of "Loading".

Issue taken from: [https://gist.github.com/zersiax/b6ab5b37cf52c8d1a6a371bf2250a872](https://gist.github.com/zersiax/b6ab5b37cf52c8d1a6a371bf2250a872)

### Steps to reproduce

- Create a new Umbraco 8 project
- Run the project from Visual Studio
- On the first screen fill in the details and click 'customize'
- On the 'Configure an ASP.Net Machine Key' screen click 'I don't want a custom Machine Key'
- On the 'Configure your database' screen leave in the default values (Microsoft SQL Server Compact (SQL CE)) and click 'continue'.
- Inspect the thumbnail for installing the starter kit - the link to accept the starter kit has the link text of "Loading"

![image](https://user-images.githubusercontent.com/4159464/65263411-82a95b80-db04-11e9-90c8-6544e920729f.png)

![image](https://user-images.githubusercontent.com/4159464/65263598-df0c7b00-db04-11e9-8763-656bd5e0195f.png)

### Expected result

The link text should say "Install starter website" and the text should be visible.

**Comment by nul800sebastiaan** [comment]
Fixed in https://github.com/umbraco/Umbraco-CMS/pull/6384
